pkg/config/default.go

```diff
@@ -299,6 +308,8 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 	c.RuntimeSupportsJSON = []string{
 		"crun",
 		"runc",
+		"kata",
+		"runsc",
 	}
 	c.RuntimeSupportsNoCgroups = []string{"crun"}
 	c.RuntimeSupportsKVM = []string{"kata", "kata-runtime", "kata-qemu", "kata-fc"}
```

Comment:
This line is inconsistent with what is provided with the file https://github.com/containers/common/blob/master/pkg/config/containers.conf#L414

Reply:
Line 414 that you provided is an empty comment line?